app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class VotesController < ApplicationController
+
+  def new
+    @vote = Vote.new
+  end
+
+  def create
+    @vote = Vote.new(vote_params)
```

Comment:
This is actually what I meant by the comment above, re: moving this into the votes controller. ++